### PR TITLE
ENH: stats.describe: add bias kw and empty array handling

### LINKED ISSUE #595: Statistics Review: describe (Trac #68)
_Original ticket http://projects.scipy.org/scipy/ticket/68 on 2006-04-02 by @rkern, assigned to @rkern._

The function `describe` in file source:trunk/Lib/stats/stats.py needs review.

Please look over the StatisticsReview guidelines and add your comments below.

### PR DESCRIPTION
**Pull request by aeklant**
closes gh-595

### CODE REVIEW

File: scipy/stats/stats.py [hunk 1]
```diff
@@ -1148,19 +1151,34 @@ def describe(a, axis=0, ddof=1):
...
+       Kurtosis (Fisher).  The kurtosis is normalized so that it is
+       zero for the normal distribution.  No degrees of freedom are used.
 
     See Also
     --------
     skew, kurtosis
 
+    Examples
+    --------
+    >>> import numpy as np
```
Comment (rgommers) [inline scipy/stats/stats.py]: This import isn't needed, it's assumed for all examples in scipy.

**Comment by rgommers** [comment]
@aeklant I edited the PR title so that it's clear what function this is about. It was a bit generic.

**Comment by rgommers** [comment]
The changes mostly look good, except for two small issues in the test:
- Can you remove the comments ``# not sure about precision with sk, skc``? Doesn't seem to make sense.
- The use of ``assert_equal`` for floating point comparisons looks off, better to replace with ``assert_allclose`` (or ``assert_almost_equal``).

**Comment by aeklant** [comment]
@ev-br default is `1e-07`. Something along the lines of `1e-13` or `1e-15`, do you think?

**Comment by rgommers** [comment]
Regarding tolerances in this case, you're replacing ``assert_equal`` so that means the match is really really good (in most cases identical). Then I'd use ``rtol=1e-15``. You can leave ``atol`` unspecified, because the default of zero is what you want.

In general, if you don't know what the tolerance should be, you can just keep on tightening it until the test fails and then back off a little. Steps of 1 order of magnitude are usually good.

**Comment by rgommers** [comment]
Last changes look good, in this goes. Thanks @aeklant, @ev-br

**Pull request merged**